autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -81,13 +81,14 @@
     <let name="launch_dummy_perception" value="true" unless="$(var scenario_simulation)"/>
     <let name="launch_dummy_vehicle" value="false" if="$(var scenario_simulation)"/>
     <let name="launch_dummy_vehicle" value="true" unless="$(var scenario_simulation)"/>
-    <let name="launch_dummy_localization" value="true"/>
+    <!-- The following modes are available as localization_sim_mode: nothing, api -->
+    <let name="localization_sim_mode" value="api"/>
```

Comment:
How about making the following change and adding `<arg name="localization_sim_mode" default="foo" description="none, foo, ..."/>` in the Optional parameters section?
```suggestion
    <let name="localization_sim_mode" value="$(var localization_sim_mode)"/>
```